README.md

```diff
@@ -10,15 +10,42 @@ Go clients for talking to a [kubernetes](http://kubernetes.io/) cluster.
 * The `transport` package is used to set up auth and start a connection.
 * The `tools/cache` package is useful for writing controllers.
 
-### Releases
+### Versioning
 
-Each top-level folder (e.g., 1.4) contains a release of clients and their dependencies.
+`client-go` will follow [semver](http://semver.org/). We will not make backwards-incompatible changes without incrementing the major version number. A branch will be created for each increase in the major version number (first digit of the version) or minor version number (second digit of the version). A new tag will be created for each increase in the patch version number (third digit of the version).
```

Comment:
We shouldn't say `digit` here.  The second digit of `10.2` is `0`.  I would leave out all three parenthetical expressions and just say

> see [semver](http://semver.org/) for definitions of major, minor, and patch.